SU-HSDO/suhumsci
# Sparkbox Sprint 5 PR

# READY FOR REVIEW

## Summary
- [STN-89](https://sparkbox.atlassian.net/browse/STN-89) - PR #466 
- [STN-99](https://sparkbox.atlassian.net/browse/STN-99) - PR #468 
- [STN-100](https://sparkbox.atlassian.net/browse/STN-100) - PR #467 
- [STN-136](https://sparkbox.atlassian.net/browse/STN-136) - PR #462 and #465 because of rebase conflicts
- [STN-141](https://sparkbox.atlassian.net/browse/STN-141) - PR #469 
- [STN-143](https://sparkbox.atlassian.net/browse/STN-143) - PR #463 

## Need Review By (Date)
asap (2/4)

## Urgency
high

## Steps to Test
- TODO: We've verified that this branch passes the code climate and circle CI tests against the develop branch.

1. See other PRs if you'd like to test locally for those steps.
2. Verify code looking at the file changes here.
3. Go into `humsci_basic` folder directory and run `npm i` and `npm run test` to verify tests pass.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
